stories/format.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Style/Format" />
+
+# Format
+
+## Number format
+
+- Use of [space] for [internationally recommended](https://en.wikipedia.org/wiki/Decimal_separator) thousands separators. No comma or point, as these are reserved for use as the decimal sign.
+- Numbers whose magnitude is less than 1, the decimal sign should be preceded by a zero.
+
+## Date format
+
+### Code Description
+
+| Code | Description                                | Example |
+| ---- | ------------------------------------------ | ------- |
+| YYYY | Year on 4 digits                           | 2021    |
+| EEE  | Weekday on 3 letters, first with uppercase | "Mon"   |
+| DD   | Day of the month with leading zero         | 08      |
+| MM   | Numerical month with leading zero          | 07      |
+| MMM  | Month abbreviation on 3 letters uppercase  | "JUL"   |
+| HH   | Hour [0-23]                                | 13      |
+| mm   | Minute with leading zero                   | 05      |
+| ss   | Second with leading zero                   | 06      |
+
+### Format list
+
+| Alias   | code                     | Example                  | Length | Context                                              |
+| ------- | ------------------------ | ------------------------ | ------ | ---------------------------------------------------- |
+| Short#1 | DD MMM                   | 20 Jul                   | 5      | Chart time axis                                      |
+| Short#2 | DDMMM HH:mm              | 20JUL 09:00              | 11     | Limited space - Dense tables, charts                 |
+| Short#3 | YYYY-MM-DD               | 2020-07-20               | 10     |                                                      |
+| Mid#1   | EEE YYYY-MM-DD HH:mm     | Mon 2020-07-20 09:00     | 20     | Checking with agenda for day of the week             |
```

Comment:
Do we really need Mid#1, not sure where we are using it currently?
I guess we should try to reduce this list as much as possible ...

Reply:
I still think that it makes sense to have the standard date with 2 formats: with and without seconds.
Space can be very scarce.